Momento/MomentoSigner.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.IdentityModel.Tokens;
+using MomentoSdk.Exceptions;
+
+namespace MomentoSdk
+{
+    public class MomentoSigner
+    {
+
+        private readonly JwtHeader jwtHeader;
```

Comment:
nit: When using the greatest of all brace styles as you are here (Allman is love), there is absolutely no reason to have leading empty newlines after your Allman { line.

Reply:
> formatting nits. Might want a linter soon.

Looks like the canonical way to do this is to bake the style-guide into the vs project and do `Edit -> Format -> Format Document`